plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState, useEffect } from 'react';
+import { useAsync } from 'react-use';
+
+import { makeStyles, Typography, Grid, Divider } from '@material-ui/core';
+import { Table, TableColumn } from '@backstage/core';
+import { FilterButton, Filters } from '../Filters';
+
+import SearchApi from '../../apis';
+
+const useStyles = makeStyles(() => ({
+  searchTerm: {
+    background: '#eee',
+    padding: '0.2%',
+    borderRadius: '10%',
+  },
+  tableHeader: {
+    marginTop: '10px',
+    display: 'flex',
+  },
+  divider: {
+    width: '1px',
+    margin: '0 20px',
+    padding: '15px 0',
```

Comment:
always use `theme.spacing(2, 0)` and so on, with integer values, for margins and paddings